code/controllers/subsystem/fire_burning.dm

```diff
@@ -32,7 +33,7 @@ SUBSYSTEM_DEF(fire_burning)
 
 		if(O.resistance_flags & ON_FIRE) //in case an object is extinguished while still in currentrun
 			if(!(O.resistance_flags & FIRE_PROOF))
-				O.take_damage(20, BURN, FIRE, 0)
+				O.take_damage(10 * delta_time, BURN, FIRE, 0)
```

Comment:
undocumented change

Reply:
Is fine. Wait = 20 / 10 is 2 * 10 is 20.